src/autopas/CMakeLists.txt

```diff
@@ -31,6 +51,11 @@ target_link_libraries(
         harmony
 )
 
+# If rules-based tuning is not disabled, also link antlr4cpp
+if (AUTOPAS_ENABLE_RULES_BASED_TUNING)
+    target_link_libraries(autopas PUBLIC antlr4cpp)
+endif()
```

Comment:
Remove these lines and put it in the `target_link_libraries` statement from above with generator statements (for consistency)

Should be something like this (can't make a suggestion because lines are too far apart):
```cmake
        $<$<BOOL:${AUTOPAS_ENABLE_RULES_BASED_TUNING}>:antlr4cpp>
```

Reply:
Done in `5044d8b5`